locales/de-de.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Was du bist? (Wähle jetzt aus, du kannst es später ändern)",
+	"select-user-type.driver": "Fahrer",
+	"select-user-type.passenger": "Passagier",
+	"request-phone.type_your_phone": "Bitte sende uns deine Telefonnummer. Wir geben diese Nummer nicht ohne Erlaubnis weiter.",
+	"request-phone.button_text": "Nummer senden",
+	"request-phone.all_set": "👌 Alles klar soweit, du kannst jetzt eine Fahrt bestellen!",
+	"passenger-index.choose_taxi_type": "Welche Art von Taxi willst du?",
+	"passenger-index.car": "🚗 Auto",
+	"passenger-index.motorbike": "🏍 Roller / Motorrad",
+	"passenger-index.s": "🔧 Einstellungen",
+	"passenger-index.error_only_known_type": "Unbekannte Auswahl, bitte wähle das Zweirad, Auto oder die Einstellungen aus.",
+	"passenger-request-location.provide_location": "Bitte teile deinen Standort, um Fahrer im Umkreis zu finden.",
+	"passenger-request-destination.provide_destination": "Wohin möchtest du? Bitte gebe die genaue Adresse an (damit der GPS-Navigator es versteht) oder beschreibe den Ort.",
```

Comment:
>(damit der GPS-Navigator es versteht)

@ro31337 Is this really necessary? It looks like you're also accepting fuzzy descriptions of the destination anyways and drivers just need to get an idea of where the passenger wants to go. Specific directions can always be shared on pickup like with a normal cab.